google/datalab/bigquery/_query.py

```diff
@@ -65,20 +67,34 @@ def __init__(self, sql, context=None, values=None, udfs=None, data_sources=None,
     self._api = _api.Api(context)
     self._data_sources = data_sources
     self._udfs = udfs
+    self._subqueries = subqueries
+    self._values = values
 
     if data_sources is None:
       data_sources = {}
 
     self._code = None
     self._imports = []
-    if values is None:
-      values = kwargs
+    if self._values is None:
+      self._values = kwargs
```

Comment:
This is going to conflict with my change. :)

Just make it self._values = {} and remove kwargs above if you want to reduce the conflict.